README.md

```diff
@@ -65,11 +64,12 @@ postgres -D data
 createdb -h localhost $USER
 
 # Finally for all methods to test that the server is reachable and to set the schema for the tests.
-psql -h localhost -f database/schema.sql
+docker build --tag gp-v2-migrations -f docker/Dockerfile.migration .
+# If you are running postgres in locally, your URL is `localhost` instead of `host.docker.internal`
+docker run -ti -e FLYWAY_URL="jdbc:postgresql://host.docker.internal/?user="$USER"&password=" -v $PWD/database/sql:/flyway/sql gp-v2-migrations migrate
```

Comment:
I am a little sad that running postgres tests locally now requires building and running a docker container. It is probably worth it over making our own migration system though. One way to make this simpler would be to still have a full schema.sql but then we would have to duplicate the information. Or maybe we can make a script that manually applies all the migration sql files. Anyway, this is fine for this PR, just thinking out loud.

Reply:
I guess u can have alternative instructions for doing it without docker

https://flywaydb.org/documentation/usage/commandline/#download-and-installation

Like using:

```bash
wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/7.5.3/flyway-commandline-7.5.3-linux-x64.tar.gz | tar xvz && sudo ln -s `pwd`/flyway-7.5.3/flyway /usr/local/bin 
```

And then:
https://flywaydb.org/documentation/getstarted/firststeps/commandline